core/modules/filter/js/filter.admin.js

```diff
@@ -7,41 +7,24 @@
 
 Backdrop.behaviors.filterStatus = {
   attach: function (context, settings) {
-    $('#filters-status-wrapper input.form-checkbox', context).once('filter-status', function () {
-      var $checkbox = $(this);
-      // Retrieve the tabledrag row belonging to this filter.
-      var $row = $('#' + $checkbox.attr('id').replace(/-status$/, '-weight'), context).closest('tr');
-      // Retrieve the vertical tab belonging to this filter.
-      var tab = $('#' + $checkbox.attr('id').replace(/-status$/, '-settings'), context).data('verticalTab');
-
-      // Bind click handler to this checkbox to conditionally show and hide the
-      // filter's tableDrag row and vertical tab pane.
-      $checkbox.bind('click.filterUpdate', function () {
-        if ($checkbox.is(':checked')) {
-          $row.show();
-          if (tab) {
-            tab.tabShow().updateSummary();
-          }
-        }
-        else {
-          $row.hide();
-          if (tab) {
-            tab.tabHide().updateSummary();
-          }
-        }
-        // Restripe table after toggling visibility of table row.
-        Backdrop.tableDrag['filter-order'].restripeTable();
-      });
+    // TableDrag is required and we should be on the filters admin page.
+    if (typeof Backdrop.tableDrag == 'undefined' || typeof Backdrop.tableDrag.filterorder == 'undefined') {
```

Comment:
If you change `filterorder` to `filter-order`, this would become
```suggestion
    if (typeof Backdrop.tableDrag == 'undefined' || typeof Backdrop.tableDrag['filter-order'] == 'undefined') {
```